code/modules/projectiles/ammo_datums.dm

```diff
@@ -230,6 +230,37 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	accurate_range = 5
 	sundering = 1.65
 
+/datum/ammo/bullet/pistol/polymer
+	name = "pistol polymer gel round"
+	icon_state = "bullet_pgr"
+	handful_icon_state = "bullet_pgr"
+	hud_state = "pistol_pgr"
+	flags_ammo_behavior = AMMO_BALLISTIC
+	damage = 20
+	penetration = 5
+	damage_type = STAMINA
```

Comment:
`damage` and `penetration` are both the same here as on `/datum/ammo/bullet/pistol`, so they don't need to be redefined.
```suggestion
	flags_ammo_behavior = AMMO_BALLISTIC
	damage_type = STAMINA
```